resources/metadata/npmExecuteScripts.yaml

```diff
@@ -0,0 +1,42 @@
+metadata:
+  name: npmExecuteScripts
+  description: Execute npm run scripts with optional install before
+  longDescription: |
+    Execute npm run scripts in all package json files, if they implement the scripts
+spec:
+  inputs:
+    params:
+      - name: install
+        type: bool
+        description: Run npm install or similar commands depending on the project structure.
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+        default: false
+      - name: runScripts
+        type: '[]string'
+        description: List of additinal run scripts to execute from package.json.
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+      - name: defaultNpmRegistry
+        type: string
+        description: URL of the npm registry to use. Defaults to https://registry.npmjs.org/
```

Comment:
Is this default contained in npm? Because I don't see it being used in the code.

Reply:
Yes. I considered adding it here as a default, but I don't see much sense in that. see here for more info https://docs.npmjs.com/using-npm/registry.html